etl_manager/meta.py

```diff
@@ -273,34 +308,48 @@ def _check_valid_data_format(self, data_format):
         if data_format not in _supported_data_formats:
             sdf = ", ".join(_supported_data_formats)
             raise ValueError(
-                f"The data_format provided ({data_format}) must match the supported data_type names: {sdf}"
+                f"The data_format provided ({data_format}) must match the supported "
+                f"data_type names: {sdf}"
             )
 
     def _check_valid_datatype(self, data_type):
         if not data_type_is_regex(data_type):
             scf = ", ".join(_supported_column_types)
             raise ValueError(
-                f"The data_type provided must match the supported data_type names: {scf}"
+                f"The data_type provided must match the supported data_type names: "
+                f"{scf}"
+            )
+
+    def _check_valid_column_sensitivity(self, sensitivity):
+        _validate_sensitivity(sensitivity)
+        if sensitivity not in _supported_column_sensitivity:
+            ss = ", ".join(_supported_column_sensitivity)
+            raise ValueError(
+                f"The sensitivity provided must match the supported "
+                f"sensitivity names: {ss}"
             )
 
     def _check_column_exists(self, column_name):
         if column_name not in self.column_names:
             cn = ", ".join(self.column_names)
             raise ValueError(
-                f"The column name: {column_name} does not match those existing in meta: {cn}"
+                f"The column name: {column_name} does not match those existing in "
+                f"meta: {cn}"
             )
 
     def _check_column_does_not_exists(self, column_name):
         if column_name in self.column_names:
             raise ValueError(
-                f"The column name provided ({column_name}) already exists table in meta."
+                f"The column name provided ({column_name}) already exists table in "
```

Comment:
Missing 'in' after 'already exists'

(also, not part of the diff, but the extra 's' at the end of this function name ('does_not_exists' rather than 'does_not_exist') could be confusing if working with this in the future. Though changing that would also mean changing anywhere that already calls this function.)